clang/lib/Format/Format.cpp

```diff
@@ -1279,6 +1279,8 @@ Style.AccessModifierOffset = -4;
   Style.UseTab = FormatStyle::UT_Always;
   Style.SpaceAfterCStyleCast = false;
   Style.SpaceAfterTemplateKeyword = false;
+  Style.SpaceBeforeParens = FormatStyle::SBPO_Haiku;
```

Comment:
you should also allow to enable this spearately from the other Haiku settings, like it is done here for BreakConstructorInitializersStyle (at line 216):

https://github.com/viveris/llvm-project/commit/18915202515159c42ceea3761ee9c22f8f5ea156#diff-c28a58077d2d1b988c339635e2fa38c9344343c64185de804c35b38c6ee9276dR216

Reply:
I have sent a commit by doing required changes. Please check.
Thanks.